src/main/cpp/src/get_json_object.hpp

```diff
@@ -30,8 +31,9 @@ namespace spark_rapids_jni {
  */
 std::unique_ptr<cudf::column> get_json_object(
   cudf::strings_column_view const& col,
-  cudf::string_scalar const& json_path,
-  spark_rapids_jni::json_parser_options options,
+  std::vector<int32_t> const& path_types,
+  std::vector<std::string> const& path_names,
+  std::vector<int64_t> const& path_indexes,
```

Comment:
If this set of values always come together, let make a separate struct for them:
```
struct path_info { // this name is just a suggestion, it can be something else if that's better
  int32_t type;
  std::string name;
  int64_t indices;
};
....
get_json_object(..., 
                             std::vector<path_info> paths,
...);
```

Reply:
Updated to `Table` now